### PR TITLE
Add initial project files

### PR DESCRIPTION
**Pull request by TheMeinerLP**
These files were extracted from the main repository for the purpose of separate updating and maintaining.

### CODE REVIEW

File: CloudFlareModule/pom.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
```
Comment (GiantTreeLP) [inline CloudFlareModule/pom.xml]: why maven assembly and not shade?

File: CloudFlareModule/src/main/java/eu/cloudnetservice/cloudflare/module/ProjectMain.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import eu.cloudnetservice.cloudflare.module.listener.ProxyAddListener;
+import eu.cloudnetservice.cloudflare.module.listener.ProxyRemoveListener;
+import eu.cloudnetservice.cloudflare.module.services.CloudFlareService;
+
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+
+public final class ProjectMain extends CoreModule {
```
Comment (GiantTreeLP) [inline CloudFlareModule/src/main/java/eu/cloudnetservice/cloudflare/module/ProjectMain.java]: we have spoken about names already

File: CloudFlareModule/src/main/java/eu/cloudnetservice/cloudflare/module/ProjectMain.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	private static ProjectMain instance;
+	private final ExecutorService executor = Executors.newSingleThreadExecutor();
```
Comment (GiantTreeLP) [inline CloudFlareModule/src/main/java/eu/cloudnetservice/cloudflare/module/ProjectMain.java]: Is another executor really necessary? The master does have it's own task scheduling built in.

File: CloudFlareModule/src/main/java/eu/cloudnetservice/cloudflare/module/listener/ProxyAddListener.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+public class ProxyAddListener implements IEventListener<ProxyAddEvent> {
+
+	@Override
+	public void onCall(ProxyAddEvent event) {
+		ProjectMain.getInstance().getExecutor().execute(new Runnable() {
+			@Override
+			public void run() {
+				CloudFlareService.getInstance().addProxy(event.getProxyServer().getProcessMeta(), ProjectMain.getInstance().getCloudFlareDatabase());
+				NetworkUtils.sleepUninterruptedly(500);
```
Comment (GiantTreeLP) [inline CloudFlareModule/src/main/java/eu/cloudnetservice/cloudflare/module/listener/ProxyAddListener.java]: What even is the reason for this weird call?

**Review by GiantTreeLP — CHANGES_REQUESTED** [review]
Please also format the code

File: CloudFlareModule/src/main/java/eu/cloudnetservice/cloudflare/module/ModuleMain.java [hunk 5]
```diff
@@ -9,28 +9,22 @@
...
 
 import java.util.Map;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
 import java.util.stream.Collectors;
 
-public final class ProjectMain extends CoreModule {
-	private static ProjectMain instance;
-	private final ExecutorService executor = Executors.newSingleThreadExecutor();
+public final class ModuleMain extends CoreModule {
```
Comment (GiantTreeLP) [inline CloudFlareModule/src/main/java/eu/cloudnetservice/cloudflare/module/ModuleMain.java]: really?

**Comment by TheMeinerLP** [comment]
I think is merge ready

**Review by GiantTreeLP — APPROVED** [review]

**Pull request merged**